IDE/ECLIPSE/DEOS/README.md

```diff
@@ -41,7 +41,7 @@ Note: To work wolfssl directory must not be under the workspace directory.
 ### Setting up a Deos project with wolfSSL
 
 The `tls_wolfssl.c` example application provides a simple function to
-run the selected examples at compile time through the following four
+run the selected test_apps at compile time through the following four
```

Comment:
Please review these changes in this file. We still point people to the same original example.